modular_skyrat/modules/supermatter_surge/code/supermatter_surge.dm

```diff
@@ -0,0 +1,40 @@
+#define SUPERMATTER_SURGE_POWER_LIMIT_UPPER 4000
+#define SUPERMATTER_SURGE_POWER_LIMIT_LOWER 200
+
+/**
+ * Supermatter Surge
+ *
+ * A very simple event designed to give engineering a challenge. It simply increases the supermatters power by a set amount, and announces it.
+ *
+ * This should be entirely fine for a powerful setup, but will require intervention on a lower power setup.
+ */
+
+/datum/round_event_control/supermatter_surge
+	name = "Supermatter Surge"
+	typepath = /datum/round_event/supermatter_surge
+	weight = 20
+	max_occurrences = 4
+	earliest_start = 10 MINUTES
```

Comment:
Increase that, it's actually not always setup by that point, and it'd just encourage engineering to make the same small setups in case there's a power surge 10 minutes in, which I don't think is your goal. I'd push this to, like, 30 minutes.

Reply:
Yeah, the event can't actually run if the supermatter isn't set up, there's a check for it

Reply:
I'd say 20 minutes is fine